RELEASENOTES.md

```diff
@@ -1 +1,2 @@
 - Hide ADO specific args from `gh gei` in favor of `gh ado2gh`.
+- `gh bbs2gh migrate-repo` now downloads the exported archive when Bitbucket's server url is provided.
```

Comment:
we're not going to include bbs2gh in our release notes until we're ready for private beta
```suggestion
```